src/lib/track-changes-utils.ts

```diff
@@ -110,3 +115,102 @@ export function getActualAttrs<T extends ManuscriptNode>(node: T) {
   }
   return attrs
 }
+
+export const getAttrsTrackingButton = (changeID: string) => {
+  const el = document.createElement('button')
+  el.className = 'attrs-popper-button'
+  el.value = changeID
+  el.innerHTML = editIcon
+
+  return el
+}
+
+const editIcon = `
+ <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M12 2L14 4L9 9L6 10L7 7L12 2Z"
+      stroke="#353535"
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M13 10V11.5C13 12.328 12.328 13 11.5 13H4.5C3.672 13 3 12.328 3 11.5V4.5C3 3.672 3.672 3 4.5 3H6"
+      stroke="#353535"
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+  </svg>
+`
+
+export const getMarkDecoration = (
+  dataTracked: TrackedAttrs,
+  htmlNode?: HTMLElement
+) => {
+  const style: {
+    background?: string
+    textDecoration?: string
+    display?: string
+  } = htmlNode?.style || {}
+  let className = undefined
+
+  const { status, operation } = dataTracked
+
+  if (
+    (operation === CHANGE_OPERATION.delete ||
+      operation === CHANGE_OPERATION.insert ||
+      operation === CHANGE_OPERATION.set_node_attributes) &&
+    status === CHANGE_STATUS.pending
+  ) {
+    style.background = '#ddf3fa'
```

Comment:
do we use this colours anywhere else? Would it make sense to assign classes here and use variables inside of css?What do you think?

Reply:
yes that makes sense, updated it to use classes.